test/unit/api/cpp/sort_black.cpp

```diff
@@ -243,6 +254,15 @@ TEST_F(TestApiBlackSort, isSequence)
   ASSERT_NO_THROW(Sort().isSequence());
 }
 
+TEST_F(TestApiBlackSort, isAbstract)
+{
+  ASSERT_TRUE(d_solver.mkAbstractSort(BITVECTOR_SORT).isAbstract());
+  // ?Array is syntax sugar for (Array ? ?)
+  ASSERT_FALSE(d_solver.mkAbstractSort(ARRAY_SORT).isAbstract());
```

Comment:
Why is this false?

Reply:
Due to the comment above, mkAbstractSort(ARRAY_SORT) returns `(ARRAY_SORT ? ?)`.

Reply:
What is the comment meant to explain here, in the context of this test assertion?